components/brave_ads/core/internal/ads/serving/choose/predict_ad_embeddings.h

```diff
@@ -0,0 +1,59 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
```

Comment:
Can we please add unit tests for this

Reply:
there are no unittests for `predict_ad` either, I guess because this is a small util and this functionality is directly tested in the serving classes?

Reply:
Oh I missed that so would be good to have tests for predict ad too as by covering here we can easily catch edge cases